src/Akeneo/Pim/Enrichment/Component/Product/Connector/FlatTranslator/FlatAttributeValueTranslator/FlatAttributeValueTranslatorInterface.php

```diff
@@ -0,0 +1,10 @@
+<?php
+
+namespace Akeneo\Pim\Enrichment\Component\Product\Connector\FlatTranslator\FlatAttributeValueTranslator;
+
+interface FlatAttributeValueTranslatorInterface
+{
+    public function support(string $attributeType, string $columnName): bool;
+
+    public function translateValues(string $attributeCode, array $properties, array $values, string $locale): array;
```

Comment:
```suggestion
    public function translate(string $attributeCode, array $properties, array $values, string $locale): array;
```
To stay consistent with the other interfaces ?